drv/lpc55-spi-server/src/main.rs

```diff
@@ -19,7 +19,7 @@ task_slot!(GPIO, gpio_driver);
 
 #[derive(Copy, Clone, PartialEq)]
 enum Trace {
-    IRQ,
+    Irq,
```

Comment:
If something is actually an acronym, I'd vote to use `#[allow(clippy::upper_case_acronyms)]` instead of messing with the capitalization

(this is what we do for `Device::IBC` in `power` and `Device::CPU` in `thermal`)

Reply:
Okay, fine by me!